### PR TITLE
Improve cacert upload check logic

### PR DESCRIPTION
**Pull request by gdams**
this currently is working (presumably due to a change in the way the jf CLI handles error handling

### CODE REVIEW

**Review by steelhead31 — APPROVED** [review]
LGTM!

File: .github/workflows/cacert-publish.yml [hunk 1]
```diff
@@ -45,9 +45,8 @@ jobs:
         run: |
           FILE=$(ls ca-certificates/debian/build/ospackage/*.deb)
           echo "File to upload: ${FILE}"
-          if jf rt s "deb/pool/main/a/adoptium-ca-certificates/$(basename $FILE)" > /dev/null; then
-            echo file_exists=true >> "$GITHUB_OUTPUT"
-          else
+          FILE_EXITS=$(jf rt s --count=true "deb/pool/main/a/adoptium-ca-certificates/$(basename $FILE)")
```
Comment (karianna) [inline .github/workflows/cacert-publish.yml:48]:
```suggestion
          FILE_EXISTS=$(jf rt s --count=true "deb/pool/main/a/adoptium-ca-certificates/$(basename $FILE)")
```

File: .github/workflows/cacert-publish.yml [hunk 2]
```diff
@@ -45,9 +45,8 @@ jobs:
...
+          if [[ "$FILE_EXITS" == "0" ]]; then
```
Comment (karianna) [inline .github/workflows/cacert-publish.yml:49]:
```suggestion
          if [[ "$FILE_EXISTS" == "0" ]]; then
```

**Review by karianna — CHANGES_REQUESTED** [review]
suggested spelling changes.

**Review by karianna — APPROVED** [review]

**Pull request merged**